jsconfig.json

```diff
@@ -5,7 +5,7 @@
     "resolveJsonModule": true,
 		"module": "commonjs",
 		"target": "ES2020",
-		"checkJs": true,
+		"checkJs": false,
```

Comment:
should this be set to false?

Reply:
It is related to files in the extension, not related to the features it provides. There are some type errors regarding AST parsing and some other functions used in the extension. To able to see actual JavaScript errors clearly, I disabled type checking for the vscode extension files for now.